models/Asset.php

```diff
@@ -213,14 +213,14 @@ class Asset extends Element\AbstractElement
      *
      * @var bool
      */
-    protected $_dataChanged = false;
+    protected $dataChanged = false;
```

Comment:
Isn't this a BC break, as the property is `protected`?

Reply:
Is a internal property